src/devices_models/devices/AC_branches.jl

```diff
@@ -914,12 +917,12 @@ function add_constraints!(
     for br in devices
         name = PSY.get_name(br)
         inv_x = 1.0 / PSY.get_x(br)
-        flow_variables_ = flow_variables[name, :]
+        flow_variables_ = @view flow_variables[name, :]
         from_bus = PSY.get_name(PSY.get_from(PSY.get_arc(br)))
         to_bus = PSY.get_name(PSY.get_to(PSY.get_arc(br)))
-        angle_variables_ = ps_angle_variables[name, :]
-        bus_angle_from = bus_angle_variables[from_bus, :]
-        bus_angle_to = bus_angle_variables[to_bus, :]
+        angle_variables_ = @view ps_angle_variables[name, :]
+        bus_angle_from = @view bus_angle_variables[from_bus, :]
+        bus_angle_to = @view bus_angle_variables[to_bus, :]
```

Comment:
not sure about these. the `@view` saves many allocations but the the iteration is worst in the loop below as there are many cache misses.